CONTRIBUTING.md

```diff
@@ -2,185 +2,79 @@
-repositories.
+The project welcomes submissions but please let everyone know what you're working on if you want to change or add something to the Gitea repositories.
 
-Before starting to write something new for the Gitea project, please
-[file an issue](https://github.com/go-gitea/gitea/issues/new).
-Significant changes must go through the [change proposal
-process](https://github.com/go-gitea/proposals) before they can be
-accepted.
+Before starting to write something new for the Gitea project, please [file an issue](https://github.com/go-gitea/gitea/issues/new). Significant changes must go through the [change proposal process](https://github.com/go-gitea/proposals) before they can be accepted.
 
-This process gives everyone a chance to validate the design, helps
-prevent duplication of effort, and ensures that the idea fits inside
-the goals for the project and tools.  It also checks that the design is
-sound before code is written; the code review tool is not the place for
-high-level discussions.
+This process gives everyone a chance to validate the design, helps prevent duplication of effort, and ensures that the idea fits inside the goals for the project and tools. It also checks that the design is sound before code is written; the code review tool is not the place for high-level discussions.
 
 ## Testing redux
 
-Before sending code out for review, run all the tests for the whole
-tree to make sure the changes don't break other usage and keep the
-compatibility on upgrade:
-
-After running for a while, the command should print
-
-```
-ALL TESTS PASSED
-```
-## Vendoring
-
-We keep a cached copy of dependencies within the `vendor/` directory,
-managing updates via [govendor](http://github.com/kardianos/govendor).
-
-Pull requests should only include `vendor/` updates if they are
-part of the same change, be it a bugfix or a feature addition.
-
-The `vendor/` update needs to be justified as part of the PR description,
-and must be verified by the reviewers and/or merger to always reference
-an existing upstream commit.
+Before sending code out for review, run all the tests for the whole tree to make sure the changes don't break other usage and keep the compatibility on upgrade. To make sure you are running the test suite exactly like we do you should install the CLI for [Drone CI](https://github.com/drone/drone) as we are using the server for continous testing, follow [these instructions](http://readme.drone.io/0.5/install/cli/). After that you can simply call `drone exec` within you working directory and it will try to run the test suite locally.
 
 ## Code review
 
-Changes to Gitea must be reviewed before they are accepted, no matter
-who makes the change even if an owners or a maintainer.  We use github's
-pull request workflow to do that and use [lgtm](http://lgtm.co) to ensure
-every PR is reviewed by at least 2 maintainers.
-
-Please try to make your pull request easy to review for us. Please read the
```

Comment:
I have prepared this file once, and the file i used for that didn't contained that paragraph -.-